package.json

```diff
@@ -94,6 +94,7 @@
     "@algoan/nestjs-google-pubsub-microservice": "file:packages/google-pubsub-microservice",
     "@algoan/nestjs-http-exception-filter": "file:packages/http-exception-filter",
     "@algoan/nestjs-logging-interceptor": "file:packages/logging-interceptor",
-    "@algoan/nestjs-pagination": "file:packages/pagination"
+    "@algoan/nestjs-pagination": "file:packages/pagination",
+    "flatted": "^3.2.9"
```

Comment:
It should be installed in the `packages/logging-interceptor/package.json` not the global `package.json`.